# Exome-seq Analysis
Exome-seq, short for exome sequencing, is a technique used in genetics research to sequence the protein-coding regions of the genome, known as the exome (Choi et al., 2019).
Exome is composed of all of the exons within the genome, the sequences which, when transcribed, remain within the mature RNA after introns are removed by RNA splicing. This includes untranslated regions of messenger RNA (mRNA), and coding regions. Exome represents only a small fraction of the entire genome {(~40Mb (coding) or 60Mb (coding + UTRs)} but contains the majority of disease-causing mutations. By focusing on the exome, we can efficiently identify genetic variants that are more likely to have functional consequences. 
<img src="[https://user-images.githubusercontent.com/link-to-your-image.png](https://github.com/user-attachments/assets/70c2b323-1fc6-4534-bbd7-08f25724d8dd)" width="200" />
Exome-seq analysis involves capturing and sequencing the exonic regions of DNA, which encode proteins. This technique allows for the identification of single nucleotide variants (SNVs), insertions, deletions and other types of genetic variations within the exome. Researchers use exome-seq to study genetic diseases, identify causative mutations, understand the genetic basis of complex traits and analyze differential gene expression, as mentioned in the context provided. Disease-related variants can be discovered by genome sequencing and related population study approaches. However, an alternative to this approach is to consider large sample sizes to detect genes or variants of importance. The potential to detect allelic variants are affected by the small to moderate sample sizes and also are insufficient to attain statistical significance and hence the variants (Wu et al. 2011). Numerous statistical approaches have been proposed for case-control studies in genomic sequence data to perform variant aggregation to study the relationship between variants and associated diseases. The aggregation tests like burden test for comparing variants count between case-control in certain genes. and variance component tests to distinguish between protective and deleterious or risk variants in a gene or both can be used to perform variant aggregation (Auer & Lettre 2015). 
